Parser/pegen.c

```diff
@@ -1,5 +1,6 @@
 #include <Python.h>
 #include <errcode.h>
+#include <errno.h>
```

Comment:
Why is this change needed?

Reply:
> 
> 
> > I was not able to find a corresponding bug logged anywhere, unfortunately. It appears that this issue has been under-reported
> 
> Please, open an issue on bpo so we can also discuss the problem there.

Got it, will close PR and open issue on bpo, I'm confident the python team would be much more well equipped to handle this than I am (It's been driving me insane so far)